grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -138,11 +138,11 @@ export const RotationForm = observer((props: RotationFormProps) => {
   const [endLess, setEndless] = useState<boolean>(shift?.until === undefined ? true : !Boolean(shift.until));
   const [rotationEnd, setRotationEnd] = useState<dayjs.Dayjs>(shiftStart.add(1, 'month'));
 
-  const [repeatEveryValue, setRepeatEveryValue] = useState<number>(1);
-  const [repeatEveryPeriod, setRepeatEveryPeriod] = useState<RepeatEveryPeriod>(RepeatEveryPeriod.DAYS);
+  const [recurrenceNum, setRecurrenceNum] = useState<number>(1);
+  const [recurrencePeriod, setRecurrencePeriod] = useState<RepeatEveryPeriod>(RepeatEveryPeriod.DAYS);
 
-  const [showActiveOnSelectedDays, setShowActiveOnSelectedDays] = useState<boolean>(false);
-  const [showActiveOnSelectedPartOfDay, setShowActiveOnSelectedPartOfDay] = useState<boolean>(false);
+  const [isMaskedByWeekdays, setIsMaskedByWeekdays] = useState<boolean>(false);
+  const [isLimitShiftEnabled, setIsLimitShiftEnabled] = useState<boolean>(false);
```

Comment:
`<boolean>` and `<number>` can be omitted as soon as the default value is provided (TS infers the type)